src/types/emr/medicationRequest.ts

```diff
@@ -66,6 +78,11 @@ export interface DosageQuantity {
   unit?: (typeof DOSAGE_UNITS)[number];
 }
 
+export interface BoundsDuration {
+  value?: number;
+  unit?: keyof typeof BOUNDS_DURATION_UNITS;
+}
+
```

Comment:
If BoundDuration is present it would be in this shape right? 🤔 

```suggestion
export interface BoundsDuration {
  value: number;
  unit: BOUNDS_DURATION_UNITS;
}

```

Reply:
@bodhish  Did you mean `typeof unit: typeof BOUNDS_DURATION_UNITS;`. We're using ` h: { label: "Hours" },` format in order to display label in the dropdown instead of d/wk/mo etc.

Reply:
Nvm. Made them mandatory